product-service/service/products.ts

```diff
@@ -0,0 +1,100 @@
+import {Product} from "../model/Product";
+
+const products: Product[] = [
```

Comment:
Please, create separate file with mocked data.

Reply:
I think it is unreasonable for an AWS Lambda to ready JSON from a file. This is a throwaway mock service. Lambdas will be getting products from DB in Task 4.